packages/ng-primitives/popover/src/popover-trigger/popover-trigger.ts

```diff
@@ -172,6 +173,16 @@ export class NgpPopoverTrigger<T = null> implements OnDestroy {
    */
   readonly open = computed(() => this.overlay()?.isOpen() ?? false);
 
+  /**
+   * Event emitted when the popover is opened.
+   */
+  readonly opened = output<void>();
+
+  /**
+   * Event emitted when the popover is closed.
+   */
+  readonly closed = output<void>();
+
```

Comment:
Could we simplify this so it is just one output that emits a boolean state. This would make it easy to bind the open state to a variable for example: `(ngpPopoverTriggerOpenChange)="isOpen = $event".

We also make all our directive inputs and outputs start with the directive selector as this helps avoid collisions, so `ngpPopoverTriggerOpenChange` would be the name.

```suggestion
  /**
   * Event emitted when the popover open state changes
   */
  readonly openChange = output<boolean>({
      alias: 'ngpPopoverTriggerOpenChange'
  });

```